src/DnsResolver/DnsResolver.Autorest/README.md

```diff
@@ -70,13 +67,14 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
-commit: 37072829f795ce840b8085035e8adf4721602f34
+commit: 6286ea80b3ac7eecfc2f230d65b2034a656f10bf
 require:
-  - $(this-folder)/../../readme.azure.noprofile.md
+  - $(this-folder)/../../../tools/SwaggerCI/readme.azure.noprofile.md
```

Comment:
This line should be the same with original

Reply:
Fixed, thank you

Reply:
Hi @Nickcandy , i am not seeing that the CI gives the error below due to the change here:

![{BBC4C6AF-4129-4998-8728-3E03B65232F7}](https://github.com/user-attachments/assets/8d4751d1-7c2a-44ef-840c-1d9af157acd0)

Would you know what the reason for this is?